tests/wpunit/Tribe/Tickets/RSVPTest.php

```diff
@@ -42,6 +51,9 @@ public function dont_die() {
 
 	public function tearDown() {
 		// your tear down methods here
+		// refresh the event ID for each test.
+		wp_delete_post( $this->event_id, true );
```

Comment:
I could be wrong here but I believe posts get deleted anyways, so this might not be needed. The unset() is though

Reply:
I may be wrong though, so don't have to do anything here

Reply:
I think you are right - but added this when it appeared there was some data bleed between tests.